README.md

```diff
@@ -8,7 +8,21 @@ Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy an
 
 Deis Monitor is changing quickly. Your feedback and participation are more than welcome, but be aware that this project is considered a work in progress.
 
-## Hacking Workflow
+## Installing with Helm
+
+Install [helm](http://helm.sh) and add the monitor repository to your chart list:
+
+```console
+$ helm repo add monitor https://github.com/monitor/charts
```

Comment:
This should be `helm repo add monitor https://github.com/deis/monitor` I assume.